packages/desktop-client/src/components/sidebar/Accounts.tsx

```diff
@@ -43,126 +36,156 @@ export function Accounts({
     (state: State) => state.account.accountsSyncing,
   );
 
-  const getAccountPath = account => `/accounts/${account.id}`;
+  const getAccountPath = (account: AccountEntity) => `/accounts/${account.id}`;
 
-  const [showClosedAccounts] = useLocalPref('ui.showClosedAccounts');
+  const [showClosedAccounts, setShowClosedAccountsPref] = useLocalPref(
+    'ui.showClosedAccounts',
+  );
 
-  function onDragChange(drag) {
+  function onDragChange(drag: { state: string }) {
     setIsDragging(drag.state === 'start');
   }
 
-  const makeDropPadding = i => {
+  const makeDropPadding = (i: number) => {
     if (i === 0) {
       return {
         paddingTop: isDragging ? 15 : 0,
         marginTop: isDragging ? -15 : 0,
       };
     }
-    return null;
+    return undefined;
+  };
+
+  async function onReorder(
+    id: string,
+    dropPos: 'top' | 'bottom',
+    targetId: unknown,
+  ) {
+    let targetIdToMove = targetId;
+    if (dropPos === 'bottom') {
+      const idx = accounts.findIndex(a => a.id === targetId) + 1;
+      targetIdToMove = idx < accounts.length ? accounts[idx].id : null;
+    }
+
+    dispatch(moveAccount(id, targetIdToMove));
+  }
```

Comment:
I am not changing targetId at this time.  It will take more work than just this file and is above the scope of this PR.  Please keep reminding me of this issue and I will try to fix it in a future PR.